docs/index.rst

```diff
@@ -172,6 +172,52 @@ uncomment the relevant configuration item(s), and insert your desired value(s).
 
   configuration
 
+
+Caching
+-------
+
+By default Astroquery employs query caching with a timeout of 1 week.
+The user can clear their cache at any time, as well as suspend cache usage,
+and change the cache location. Caching persists between Astroquery sessions.
+If you know the service you are using has released new data recently, or if you believe you are
+not recieving the newest data, try clearing the cache.
+
+
+The Astroquery cache location is divided by service, so each service's cache should be managed invidually,
+however whether the cache is active and the expiration time are controlled centrally through the
+astroquery ``cache_conf`` module. Astroquery uses the Astropy configuration infrastructure, information about
+temporarily or permanently changing configuration values can be found
+`here <https://docs.astropy.org/en/latest/config/index.html>`_.
+
+Shown here are the cache properties, using Simbad as an example:
+
+.. code-block:: python
```

Comment:
do we want to cover this with the remote tests. However, if this is intentionally skipped, please indicate it with doctest-skip instead

Reply:
I didn't think  remote data is actually required for this, so I didn't give the remote data directive.  I thought that unless explicitely skipped the tests would run under doctesting (once the file is added, right now the whole file is skipped at the top).

Reply:
hmm, OK, fair enough. The file has opted into testing on `main` already, so maybe this will just need a rebase to make sure these blocks pass.